proposals/0293-vehicle-type-filter.md

```diff
@@ -266,22 +144,176 @@ The Android proxy will need to implement the above protocol changes. In addition
 #### Metadata for Android Router Service
 
 1. This resource file needs to be referenced in the manifest file of the project as `metadata` for Android Router Service.
-2. When the proxy receives `GetVehicleTypeACK`, it will check if the vehicle type information is defined in the provided supported vehicle type list.
+2. When the proxy receives `StartServiceACK` with vehicle type details, it will check if the vehicle type information is defined in the provided supported vehicle type list.
 3. If the vehicle filter is not defined, the proxy can skip the check. In this case, the Android Router Service is supported for all SDL enabled systems.
 
-#### Determining Vehicle Type Info
+#### Flowcharts
+
+1. In the current implementation, all SDL apps need to start version negotiations and register themselves on the SDL enabled system. This proposal recommends starting the RPC service by the SDL Device Listener to get vehicle type information from an SDL enabled IVI system before notifying clients about an SDL connection.
+2. The proxy determines the appropriate router service to deploy based on vehicle type information received in the `StartServiceACK` protocol message. The proxy will check the metadata of the supported vehicle type list to determine the router service to deploy. The deployed router service will also receive vehicle type information. Once the Android Router service is started, it will notify the client and provide vehicle type information. 
```

Comment:
```suggestion
2. The proxy determines the appropriate router service to deploy based on vehicle type information received in the `StartServiceACK` protocol message. The proxy will check the metadata of the supported vehicle type list to determine the router service to deploy. The deployed router service will also receive vehicle type information. Once the Android Router Service is started, it will notify the client and provide vehicle type information. 
```